packages/audio-block/src/AudioBlock.spec.ct.tsx

```diff
@@ -0,0 +1,151 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from 'cypress/react';
+import { AssetDummy, withAppBridgeBlockStubs } from '@frontify/app-bridge';
+import { AudioBlock } from './AudioBlock';
+import { TextPosition } from './types';
+import { AUDIO_ID } from './settings';
+const AudioBlockSelector = '[data-test-id="audio-block"]';
+const AudioTagSelector = '[data-test-id="audio-block-audio-tag"]';
+const UploadPlaceholderSelector = '[data-test-id="upload-placeholder"]';
+const ItemToolbarSelector = '[data-test-id="item-toolbar"]';
+
+describe('Audio Block', () => {
+    it('renders an audio block', () => {
+        const [AudioBlockWithStubs] = withAppBridgeBlockStubs(AudioBlock);
+
+        mount(<AudioBlockWithStubs />);
+        cy.get(AudioBlockSelector).should('exist');
+    });
+
```

Comment:
Not needed as the other tests take care of that
```suggestion
```

Reply:
remove within [commit](https://github.com/Frontify/guideline-blocks/pull/475/commits/d5db1bb55113c9f03cf16bd62101414ab00bc979)